code/modules/goonhub/api/api.dm

```diff
@@ -163,7 +163,7 @@ var/global/datum/apiHandler/apiHandler
 
 		// Validation
 		var/datum/apiModel/model = new route.correct_response
-		if (istype(model, /datum/apiModel/Paginated))
+		if (istype(model, /datum/apiModel/Paginated) || istype(model, /datum/apiModel/Message))
```

Comment:
Sneaking this in as a fix for message responses (not explicitly related to bans)